notebooks/polynomial_regression.livemd

```diff
@@ -0,0 +1,157 @@
+# defn transform
```

Comment:
We should either convert this into a guide (see the others as an example) or remove this notebook. :)

Reply:
Could it be better to update the linear regression guide with polynomial regression also in it? Instead of having one with just polynomial?